app.js

```diff
@@ -11,8 +11,13 @@ const i18nBackend = require('i18next-node-fs-backend');
 const os = require('os');
```

Comment:
Setting still available in single display mode
![image](https://user-images.githubusercontent.com/2982784/104134344-c8afb180-533d-11eb-9418-aa106c593eaa.png)